src/components/Link/Link.scss

```diff
@@ -13,4 +13,24 @@
         color: var(--link-color-hover);
         text-decoration: var(--link-decoration-hover);
     }
+
+    &--context {
+        @each $context in (brand, neutral, primary, accent, info, good, warning, bad) {
+            &_#{ $context } {
+                color: var(--color-#{$context});
+
+                &:hover {
+                    color: var(--color-#{$context}-40);
+                }
+            }
+        }
+
+        &_muted {
+            color: var(--color-muted);
+
+            &:hover {
+                color: var(--color-neutral);
```

Comment:
Why is there no `--color-muted-40` :thinking: , even it literally is defined as `--color-neutral`?

@carlobernardini Perhaps it's even better to define a `--color-#{$context}-hover` to have all `:hover` colors consistent?